src/zql/ast-to-ivm/pipeline-builder.ts

```diff
@@ -233,7 +233,8 @@ function applyFullTableAggregation<T extends Entity>(
   stream: DifferenceStream<T>,
   aggregations: Aggregation[],
 ) {
-  let ret = stream;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let ret: DifferenceStream<any> = stream;
```

Comment:
Could this be?

```suggestion
  let ret: DifferenceStream<number | Entity> = stream;
```

Reply:
This type error confounded me. Still trying to figure it out and also why it wasn't a problem until I removed the `reader` and `writer` indirections.